packages/pds/src/sql-repo-storage.ts

```diff
@@ -237,9 +237,6 @@ export class SqlRepoStorage extends RepoStorage {
           root: cid.toString(),
           indexedAt: this.getTimestamp(),
         })
-        .onConflict((oc) =>
-          oc.column('did').doUpdateSet({ root: cid.toString() }),
-        )
```

Comment:
So the `prev` here isn't necessarily the `prev` listed in the commit, it could also reference the commit being replaced in a rebase, right?  Just wanted to double-check, since I could imagine this on-conflict handling rebases.

Reply:
I think it's all good, just wanted to double-check 👍